src/components/dms/dialogs/NewChatDialog.tsx

```diff
@@ -31,7 +32,8 @@ export function NewChat({
       logEvent('chat:open', {logContext: 'NewChatDialog'})
     },
     onError: error => {
-      Toast.show(error.message)
+      logger.error('Failed to create chat', {message: error})
```

Comment:
i think this needs to be `safeMessage`

Reply:
Seems like it's super inconsistent across the whole codebase, I'll change it here but we should look at it in other places too